src/darwin/Framework/CHIP/MTRMetricKeys.h

```diff
@@ -0,0 +1,56 @@
+/**
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <lib/core/CHIPError.h>
+#include <tracing/metric_event.h>
+
+namespace chip {
+namespace Tracing {
+namespace DarwinFramework {
+
+constexpr Tracing::MetricKey kMetricDeviceCommissioning = "dwnfw_device_commissioning";
```

Comment:
These keys get exposed to API consumers, effectively, right?  Should there be something about "matter" in them?